lib/HelpView.mli

```diff
@@ -0,0 +1,13 @@
+(** This module provides UI components and attributes for a help panel in a Nottui-based application. *)
+
+val help_visible : bool Lwd.var
+(** [help_visible] is a variable indicating whether the help panel is currently visible. *)
+
+val help_panel : Nottui.ui
+(** [help_panel] constructs the user interface for the help panel. This includes various help-related commands and their corresponding visual attributes, making it easy for users to understand available interactions. *)
+
+val help_keyboard_area : Nottui.ui
+(** [help_keyboard_area] provides a keyboard interaction area where pressing 'q' will close the help panel. *)
```

Comment:
This value is not used.

The problem would not be solved by simply removing the value. In my opinion, the problem is more that the logic for opening/closing the help panel is spread between two very different location: `HelpView.ml` and `InteractiveViewer.ml`. This logic should be in one place!

Reply:
Move help_visible variable into InteractiveViewer.ml